diesel_cli/src/setup_sql/postgres/initial_setup/up.sql

```diff
@@ -0,0 +1,36 @@
+-- This file was automatically created by Diesel to setup helper functions
+-- and other internal bookkeeping. This file is safe to edit, any future
+-- changes will be added to existing projects as new migrations.
```

Comment:
[no](https://www.merriam-webster.com/dictionary/setup) and [no](https://www.merriam-webster.com/dictionary/bookkeeping) English is about as consistent as PHP